src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -39,14 +41,16 @@ pub fn summarize_level_deltas(level_deltas: &LevelDeltas) -> LevelDeltasSummary
     let mut insert_sub_level_id = u64::MAX;
     let mut insert_table_infos = vec![];
     for level_delta in &level_deltas.level_deltas {
-        if !level_delta.removed_table_ids.is_empty() {
-            delete_sst_levels.push(level_delta.level_idx);
-            delete_sst_ids_set.extend(level_delta.removed_table_ids.iter().clone());
-        }
-        if !level_delta.inserted_table_infos.is_empty() {
-            insert_sst_level_id = level_delta.level_idx;
-            insert_sub_level_id = level_delta.l0_sub_level_id;
-            insert_table_infos.extend(level_delta.inserted_table_infos.iter().cloned());
+        if let DeltaType::IntraLevel(intra_level) = level_delta.get_delta_type().unwrap() {
```

Comment:
> Shall we add GroupConstruct and GroupDestroy in LevelDeltasSummary? So that someone is less likely to appy IntraLevel without constructing groups beforehand and destroying groups afterwards.

fixed